tests/test_ops/test_fused_bias_leakyrelu.py

```diff
@@ -2,48 +2,93 @@
             return
-        cls.input_tensor = torch.randn((2, 2, 2, 2), requires_grad=True).cuda()
-        cls.bias = torch.zeros(2, requires_grad=True).cuda()
+        if IS_CUDA_AVAILABLE:
+            cls.input_tensor = torch.randn((2, 2, 2, 2),
+                                           requires_grad=True).cuda()
+            cls.bias = torch.zeros(2, requires_grad=True).cuda()
+        elif IS_NPU_AVAILABLE:
+            cls.input_tensor = torch.randn((2, 2, 2, 2),
+                                           requires_grad=True).npu()
+            cls.bias = torch.zeros(2, requires_grad=True).npu()
 
-    @pytest.mark.skipif(not torch.cuda.is_available(), reason='requires cuda')
-    def test_gradient(self):
+    @pytest.mark.parametrize('device', [
+        pytest.param(
+            'cuda',
+            marks=pytest.mark.skipif(
+                not IS_CUDA_AVAILABLE, reason='requires CUDA support')),
+        pytest.param(
+            'npu',
+            marks=pytest.mark.skipif(
+                not IS_NPU_AVAILABLE, reason='requires NPU support'))
+    ])
+    def test_gradient(self, device):
 
         from mmcv.ops import FusedBiasLeakyReLU
         if _USING_PARROTS:
+            if IS_CUDA_AVAILABLE:
+                gradcheck(
+                    FusedBiasLeakyReLU(2).cuda(),
+                    self.input_tensor,
+                    delta=1e-4,
+                    pt_atol=1e-3)
+            elif IS_NPU_AVAILABLE:
+                gradcheck(
+                    FusedBiasLeakyReLU(2).npu(),
+                    self.input_tensor,
+                    delta=1e-4,
+                    pt_atol=1e-3)
+        else:
+            if IS_CUDA_AVAILABLE:
+                gradcheck(
+                    FusedBiasLeakyReLU(2).cuda(),
+                    self.input_tensor,
+                    eps=1e-4,
+                    atol=1e-3)
+            elif IS_NPU_AVAILABLE:
+                gradcheck(
+                    FusedBiasLeakyReLU(2).npu(),
+                    self.input_tensor,
+                    eps=1e-4,
+                    atol=1e-3)
+
+    @pytest.mark.parametrize('device', [
+        pytest.param(
+            'cuda',
+            marks=pytest.mark.skipif(
+                not IS_CUDA_AVAILABLE, reason='requires CUDA support')),
+        pytest.param(
+            'npu',
+            marks=pytest.mark.skipif(
+                not IS_NPU_AVAILABLE, reason='requires NPU support'))
+    ])
+    def test_gradgradient(self, device):
+
+        from mmcv.ops import FusedBiasLeakyReLU
+        if IS_CUDA_AVAILABLE:
             gradcheck(
                 FusedBiasLeakyReLU(2).cuda(),
                 self.input_tensor,
-                delta=1e-4,
-                pt_atol=1e-3)
-        else:
+                eps=1e-4,
+                atol=1e-3)
+        elif IS_NPU_AVAILABLE:
             gradcheck(
-                FusedBiasLeakyReLU(2).cuda(),
+                FusedBiasLeakyReLU(2).npu(),
```

Comment:
Same as the above comment that merges them to one case using `.to(device)` method.